packages/addon/src/background.ts

```diff
@@ -195,12 +207,19 @@ const THUNDERMAIL_DISPLAY_NAME = 'Thundermail';
 browser.runtime.onMessage.addListener(async (message) => {
   const { email, name, token } = message;
   switch (message.type) {
-    case 'TB/PING':
+    case PING:
       console.log('[background] got the ping from the bridge');
-      console.log(message);
       break;
 
-    case 'TB/OIDC_TOKEN':
+    case OIDC_USER:
+      await browser.storage.local.set({
+        [STORAGE_KEY_AUTH]: message.user,
+      });
+
+      console.log(`🎯 user auth stored in add-on context.`);
+      break;
```

Comment:
We save the auth cookie (sent by the bridge) so that it's available to other parts of the tb.pro add-on.

Note: we use `browser.storage.local` to do this. If we used `localStorage`, the data would not be available outside of `background.ts`.